src/server/language.json

```diff
@@ -215,7 +215,6 @@
 			"Node": "Node",
 			"NodeNadir": "NodeNadir",
 			"NodePeak": "NodePeak",
-			"None": "Keine",
```

Comment:
There is two "None" in the json file that makes it invalide, so I deleted one of them.